### PR TITLE
Feature/taalhuis participants files view

### PR DESCRIPTION
**Pull request by bbawuah**
This branch contains 

- the route for the participants files
- the event detail fieldset
- the event detail list item

### CODE REVIEW

File: client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Files/ParticipantsFilesView.tsx [hunk 1]
```diff
@@ -5,18 +5,36 @@ import Headline, { SpacingType } from 'components/Chrome/Headline'
...
+            <TaalhuisParticipantsDetailTabs activeTabId={Tabs.Intake} routeState={routeState} />
+            <EventDetailFieldset
+                type={EventDetailTypes.intake}
+                readOnly={true}
+                description={i18n._(
+                    t`Proin imperdiet mauris eget gravida faucibus. In sed venenatis elit. 
+                    Praesent viverra eleifend quam quis mattis. Duis vitae volutpat lorem, ac eleifend nunc. 
+                    Praesent quis tellus ac nulla sodales lacinia. Donec tempor odio neque, at egestas sem imperdiet eu. 
+                    In sed molestie ex, non efficitur dolor.`
+                )}
```
Comment (RickWoltheus) [inline client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Files/ParticipantsFilesView.tsx:36]: this should be dynamic data right? if yes then, translating it is not necessary if this is just stub data

File: client/src/components/Domain/Files/Fieldsets/EventDetailFieldset.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                            <Paragraph className={styles.sectionTitle}>{i18n._(t`Omschrijving`)}</Paragraph>
+                            <Paragraph>{description}</Paragraph>
+                            <Paragraph className={styles.sectionTitle}>{i18n._(t`Checklist`)}</Paragraph>
+                            <div className={styles.containerList}>
+                                <ul>
+                                    <li>Welk aanbod volg je?</li>
+                                    <li>Past het aanbod bij wat je wil leren?</li>
+                                    <li>Ben je tevreden over wat je leert?</li>
+                                    <li>Gebruik je wat je leert al in je dagelijks leven?</li>
+                                    <li>Zou je wat anders willen leren?</li>
```
Comment (RickWoltheus) [inline client/src/components/Domain/Files/Fieldsets/EventDetailFieldset.tsx:100]: translations?

**Review by RickWoltheus — CHANGES_REQUESTED** [review]

File: client/src/components/Domain/Files/Fieldsets/EventDetailFieldset.tsx (hunk 2 above)
Comment (bbawuah) [reply client/src/components/Domain/Files/Fieldsets/EventDetailFieldset.tsx:100]: fixed it in this pr https://github.com/lifelynl/bisc-taalhuizen/pull/226

File: client/src/components/Domain/Files/Table/FilesEventsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        [MonthAbbreviations.mrt]: i18n._(t`mrt`),
+        [MonthAbbreviations.apr]: i18n._(t`apr`),
+        [MonthAbbreviations.mei]: i18n._(t`mei`),
+        [MonthAbbreviations.jun]: i18n._(t`jun`),
+        [MonthAbbreviations.jul]: i18n._(t`jul`),
+        [MonthAbbreviations.aug]: i18n._(t`aug`),
+        [MonthAbbreviations.sep]: i18n._(t`sep`),
+        [MonthAbbreviations.okt]: i18n._(t`okt`),
+        [MonthAbbreviations.nov]: i18n._(t`nov`),
+        [MonthAbbreviations.dec]: i18n._(t`dec`),
```
Comment (bbawuah) [inline client/src/components/Domain/Files/Table/FilesEventsTable.tsx:49]: Not sure if this is the right solution for abbreviations

**Review by RickWoltheus — APPROVED** [review]
i will approve this, just check my last comment i added

**Pull request merged**